src/pages/workspace/taxes/WorkspaceTaxesPage.tsx

```diff
@@ -0,0 +1,195 @@
+    );
+
+    const taxesList = useMemo<ListItem[]>(
+        () =>
+            Object.entries(policy?.taxRates?.taxes ?? {})
+                .map(([key, value]) => ({
+                    text: value.name,
+                    alternateText: textForDefault(key),
+                    keyForList: key,
+                    isSelected: !!selectedTaxesIDs.includes(key),
+                    isDisabledCheckbox: key === defaultExternalID,
+                    rightElement: (
+                        <View style={styles.flexRow}>
+                            <Text style={[styles.disabledText, styles.alignSelfCenter]}>
+                                {value.isDisabled ? translate('workspace.common.disabled') : translate('workspace.common.enabled')}
+                            </Text>
+                            <View style={[styles.p1, styles.pl2]}>
+                                <Icon
+                                    src={Expensicons.ArrowRight}
+                                    fill={theme.icon}
+                                />
+                            </View>
+                        </View>
+                    ),
+                }))
+                .sort((a, b) => a.text.localeCompare(b.text)),
+        [policy?.taxRates?.taxes, textForDefault, defaultExternalID, selectedTaxesIDs, styles, theme.icon, translate],
+    );
+
+    const isLoading = !isOffline && taxesList === undefined;
+
+    const toggleTax = (tax: ListItem) => {
+        const key = tax.keyForList;
+        if (typeof key !== 'string') {
+            return;
+        }
+
+        setSelectedTaxesIDs((prev) => {
+            if (prev?.includes(key)) {
+                return prev.filter((item) => item !== key);
+            }
+            return [...prev, key];
+        });
+    };
+
+    const toggleAllTaxes = () => {
+        const taxesToSelect = taxesList.filter((tax) => tax.keyForList !== defaultExternalID);
+        setSelectedTaxesIDs((prev) => {
+            if (prev.length === taxesToSelect.length) {
+                return [];
+            }
+
+            return taxesToSelect.map((item) => (item.keyForList ? item.keyForList : ''));
+        });
+    };
+
+    const getCustomListHeader = () => (
+        <View style={[styles.flex1, styles.flexRow, styles.justifyContentBetween, styles.pl3, styles.pr9]}>
+            <Text style={styles.searchInputStyle}>{translate('common.name')}</Text>
+            <Text style={[styles.searchInputStyle, styles.textAlignCenter]}>{translate('statusPage.status')}</Text>
+        </View>
+    );
+
+    const headerButtons = (
+        <View style={[styles.w100, styles.flexRow, isSmallScreenWidth && styles.mb3]}>
+            <Button
+                medium
+                success
+                onPress={() => {}}
+                icon={Expensicons.Plus}
+                text={translate('workspace.taxes.addRate')}
+                style={[styles.mr3, isSmallScreenWidth && styles.w50]}
```

Comment:
In other places, we’re using mr2 or gap2 between buttons on headers, which is causing [this issue](https://github.com/Expensify/App/issues/49786).